packages/@okta/vuepress-site/docs/guides/pwd-optional-change-email/main/react/integrationsteps.md

```diff
@@ -0,0 +1,139 @@
+
+<div class="half border">
+
+![Screenshot showing an edit link next to the primary email address.](/img/pwd-optional/pwd-optional-change-email-my-account-js-react-edit-email.png)
+
+</div>
+
+### 3. The user submits a new primary email
+
+When the user clicks on the **Edit** link, display a dialog that allows the user to change and submit their new email address.
+
+<div class="half border">
+
+![Screenshot showing a page with a new primary email input field and continue button.](/img/pwd-optional/pwd-optional-change-email-my-account-js-react-submit-email.png)
+
+</div>
+
+1. When the user clicks **Continue** and submits their new email address, create an object of type [`AddEmailPayload`](https://github.com/okta/okta-auth-js/blob/master/docs/myaccount/modules.md#addemailpayload) and set
+
+* `profile.email` to the new primary email.
+* `role` to `PRIMARY` which identifies the email as the primary email address.
+* `sendEmail` to `true` which sends the email challenge to the newly added email. The default is `true`.
+
+```json
+{
+   "profile":{
+      "email":"john.doe.new@example.com"
+   },
+   "sendEmail":true,
+   "role":"PRIMARY"
+}
+```
+
+2. Call [`addEmail`](https://github.com/okta/okta-auth-js/blob/master/docs/myaccount/modules.md#addemail) and pass in the new `AddEmailPayload` object.
+
+```javascript
+import {
+  addEmail
+} from '@okta/okta-auth-js/myaccount';
+
+const handleAddEmail = async (role, email) => {
+  return addEmail(oktaAuth, {
+    payload: {
+      profile: {
+        email
+      },
+      sendEmail: true,
+      role
+    }
+  });
+};
+```
+
+### 4. Your app handles email verification response
+
+`addEmail` returns a promise that contains an [`EmailTransaction`](https://github.com/okta/okta-auth-js/blob/master/docs/myaccount/classes/EmailTransaction.md) object. The object has a `status` of `UNVERIFIED` that indicates that the user needs to verify their identity with the new email.
+
+```json
+{
+   "id":"2e355d8376d4a12bb45ed54ebf8dd4d5",
+   "status":"UNVERIFIED",
+   "profile":{
+      "email":"robnicolo+oie-2022-11a@gmail.com"
+   },
+   "roles":[
+      "PRIMARY"
+   ],
+}
+```
+
+Build the logic that handles this response and sends the user to a dialog where they enter the OTP.
+
+### 5. The user verifies their identity with the new email
+
+The user opens the email sent by Identity Engine and copies the One-Time Passcode (OTP) to your app. Create a dialog to allow the user to enter and submit the OTP.
+
+<div class="half border">
+
+![Screenshot showing a page with a new primary email input field and continue button.](/img/pwd-optional/pwd-optional-change-email-my-account-js-react-verify-email.png)
+
+</div>
+
+1. When the user submits the OTP, create an object of type [`VerificationPayload`](https://github.com/okta/okta-auth-js/blob/master/docs/myaccount/modules.md#VerificationPayload) and set `verificationCode` to the OTP entered by the user.
```

Comment:
I don't think this needs to be a numbered list as the code isn't really doing these two steps separately. (See my comments in step 3 which perhaps tries to make the code do it in two steps by comparison).  Probably easiest to discuss this one.

Reply:
numbers removed.